doc/mr.txt

```diff
@@ -36,6 +38,52 @@ Get |List| of MRU/MRW/MRR by calling |mr#mru#list()|, |mr#mrw#list()|, or
 	:echo mr#mrr#list()
 <
 
+How to use under fuzzy finder or auto completion plugin.
+
+CTRLP USAGE						*mr-usage-ctrlp*
+
+|ctrlp| is Pure Vim script fuzzy finder plugin.
```

Comment:
Please do NOT refer other plugins that might not be installed on user's system. Instead, please add URL of the plugin.

Reply:
Sure, remove content link and add URL link.